src/drive/web/modules/filelist/AddFolder.jsx

```diff
@@ -58,6 +59,19 @@ const AddFolder = ({
           >
             —
           </TableCell>
+          {!isMobile &&
+            additionalColumns &&
+            additionalColumns.map(column => (
+              <TableCell
+                key={column.label}
+                className={cx(
+                  styles['fil-content-cell'],
+                  styles['fil-content-certification']
+                )}
+              >
+                —
+              </TableCell>
+            ))}
```

Comment:
Maybe put this into the column spec. 

emptyContent: () => <TableCell ... />

Reply:
J'ai préféré créer un nouveau composant EmptyCell et l'utiliser dans AddFolder partout ou c'était nécessaire.

Concernant le sujet des colonnes, ça donne ça au final : 
```
{extraColumns.map(() => (
    <EmptyCell className={styles['fil-content-narrow']} />
))}
```